spec/controllers/reservations_controller_spec.rb

```diff
@@ -453,58 +404,57 @@
   end
 
   describe '#edit (GET /reservations/:id/edit)' do
-    ## Unhappy paths
+    # SMELLS:
+    #   - message chain
     it_behaves_like 'inaccessible by banned user' do
-      before { get 'edit', id: @reservation.id }
+      before { get 'edit', id: 1 }
     end
 
     context 'when accessed by patron' do
       before(:each) do
-        sign_in @user
-        get 'edit', id: @reservation.id
+        mock_user_sign_in
+        get 'edit', id: 1
       end
-      include_examples 'cannot access page'
+      include_examples 'redirected request'
     end
 
     context 'when accessed by checkout person disallowed by settings' do
       before(:each) do
-        sign_in @checkout_person
-        allow(@ac).to receive(:checkout_persons_can_edit).and_return(false)
-        get 'edit', id: @reservation.id
```

Comment:
Well, here we don't actually have a reservation created because it should redirect before trying to get the reservation in question. 

`get :edit` requires an id to start the request, so I just used 1.